compiler/damlc/tests/daml-test-files/ProposalIou.EXPECTED.md

```diff
@@ -18,8 +18,9 @@
 >   
 >   (no fields)
 
-## Template Instances
+## Data Types
```

Comment:
@fmota-da I'm not sure what is expected here - can you enlighten me? :)

Reply:
It is `## Template Instances`. This would almost certainly be solved by #2239.

Reply:
We can do this in a separate PR but before start telling the world that generic templates have landed.

Reply:
👍

Reply:
Ok! Thanks for the info